### PR TITLE
Add pipeline running validations

### PR DESCRIPTION
**Pull request by jmar910**
### Description
Add a pipeline running validation to prevent the user from updating transforms or editing pipeline settings while the pipeline is running

Fixes https://github.com/ConduitIO/conduit/issues/145

### Screenshots
Screen Shot 2022-04-01 at 1 58 31 PM
Screen Shot 2022-04-01 at 1 58 35 PM
Screen Shot 2022-04-01 at 1 58 39 PM

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

**Review by lovromazgon — COMMENTED** [review]
Looks good, how hard would it be adding the same check for editing a connector? Right now I can get to the edit connector popup and press "update" but nothing happens.

EDIT: Actually I see a popup warning that the pipeline is running. Something strange happens if I try to delete the connector first though, then the update doesn't do anything anymore and looks like it's broken.

**Comment by jmar910** [comment]
@lovromazgon resolved!

File: ui/config/environment.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ module.exports = function (environment) {
 
   if (environment === 'development') {
     ENV['ember-cli-mirage'] = {
-      enabled: true,
+      enabled: ENV.isDevMirageEnabled,
```
Comment (jayjayjpg) [inline ui/config/environment.js:34]: Does this require any documentation or do most users run the app directly against the conduit API anyways?

File: ui/tests/acceptance/pipeline/index-test.js [hunk 2]
```diff
@@ -327,12 +339,34 @@ module('Acceptance | pipeline/index', function (hooks) {
...
-    test('does not show a degraded button', async function () {
+    test('does not show a degraded button', function () {
       assert.dom(page.pipelineDegradedButton).doesNotExist();
     });
+
+    test('it disables adding connectors', function () {
+      assert.dom(page.pipelineAddNewNodeDisabledButton).exists();
+    });
+
+    test('it disables updating transforms', async function () {
```
Comment (jayjayjpg) [inline ui/tests/acceptance/pipeline/index-test.js:358]: Neat! :sparkles:

**Review by jayjayjpg — APPROVED** [review]
Awesome work as always, this reads great! :sparkles:

File: ui/config/environment.js (hunk 1 above)
Comment (jayjayjpg) [reply ui/config/environment.js:34]: Not a blocker, just curious by the way! :-)

Comment (jmar910) [reply ui/config/environment.js:34]: No documentation required. We originally ran it in passthru mode anyway by default, so this is the same thing, but just fully disables mirage. Most if not all users should be running the app directly against the conduit API

Comment (jayjayjpg) [reply ui/config/environment.js:34]: Neat and thank you for the outline!

**Pull request merged**